committee-code-of-conduct/election.md

```diff
@@ -0,0 +1,80 @@
+# Kubernetes Code of Conduct Committee Elections
+
+This document outlines the process, for Code of Conduct Committee (CoCC) elections.
+
+### Length of Terms
+
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed for 2 year terms and the other 2 members will be appointed for a 1 year term.
+* In all subsequent elections the term will be for 2 years.
+* In alternating years either 2 or 3 seats will be up for election as terms expire.
+
+### Eligibility for voting
+
+* Voting will be done by members of the steering committee.
```

Comment:
From https://github.com/kubernetes/community/pull/3912/files#r309490546:

> I read this as saying steering committee membership does not prevent code of conduct committee membership (or vice-versa).

In this case, to cover all bases, we could perhaps add a line or two about recusal from voting in case a Steering Committee member is running for the CoCC election?